resources/test/heads/test/doctopus-test.edn

```diff
@@ -1,6 +1,6 @@
 {:output-root "docs"
  :html-commands ["lein marg -f index.html"]
  :name "doctopus-test"
- :source-control :git
+ :source-control "git"
```

Comment:
i had considered possibly doing the `prepare` / `transform` dance to make these keywords, thoughts?